Adapt links (#303)
SoftwareAG/c8y-releasenotes: content/release-10-18-0/announcements-10-18-0.md

### REST API changes

#### Implemented

##### Breaking change in the Alarms, Events, Measurements APIs - required parameters will be introduced

> As announced earlier, see [release 10.17](/release-10-17-0/announcements-10-17-0), at least one query parameter limiting the affected data will be required to prevent accidental deletion of too many objects during a bulk delete operation.

Reviewer: suggestion: "As announced with [release 10.17](..."
avoids a comma and feels more flowy

Author: It has not only been announced with 10.17 but also earlier, which is why I used this phrasing. I use "with 10.17" if it has been announced there for the first time. Anyway, I will no longer use this duplicate announcements in the future.

##### Breaking change in all REST APIs

As announced earlier, see [release 10.17](/release-10-17-0/announcements-10-17-0), wrong values for `pageSize` or `currentPage` now result in a 422 (Unprocessable entity) HTTP status code instead of a 500 (Internal server error) HTTP status code.

### Microservice infrastructure changes

#### Planned

##### Breaking change: Use of Linux cgroup v2

> As announced earlier, see [release 10.17](/release-10-17-0/announcements-10-17-0), in Cumulocity IoT versions higher than 10.18, microservices must use a Linux cgroup v2 aware application runtime. When executing microservices which are not compatible with cgroup v2 on Cumulocity IoT in these versions it might happen that the information provided by the application runtime concerning available CPU and memory is not correct. This might lead to incorrect memory and thread allocation in the microservice container process.

Author: ```suggestion
As announced earlier, see [release 10.17](/release-10-17-0/announcements-10-17-0), in Cumulocity IoT versions after 10.18, microservices must use a Linux cgroup v2 aware application runtime. When executing microservices which are not compatible with cgroup v2 on Cumulocity IoT in these versions it might happen that the information provided by the application runtime concerning available CPU and memory is not correct. This might lead to incorrect memory and thread allocation in the microservice container process.
```

All of the impacted injection tokens have been marked as deprecated with 10.18, and references to their replacements are provided in the respective documentation and deprecation notice.

#### Implemented

##### Breaking change in the Map widget

> As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), the "Map" widget has been migrated and support for real-time updates on all devices has been replaced by a configurable refresh interval. This change allows us to display more then 100 devices on the map by default. Additionally, the "Map" widget no longer supports the "show track" option.

Reviewer: oh, here it uses the wording :)

Author: see comment above, slightly different situation :-)
